circe/src/main/scala/io/finch/circe/Printers.scala

```diff
@@ -0,0 +1,42 @@
+package io.finch.circe
+
+import java.nio.charset.{Charset, StandardCharsets}
+
+import com.twitter.io.Buf
+import io.circe.{Json, Printer}
+
+trait Printers {
```

Comment:
I don't really understand why we need to extract this? is that because we want to support `io.finch.circe.accumulating.dropNullValues._` (in the meaning of accumulative decoding and a special printing)?

Maybe it will actually be fine to just start with `io.circe.accumulating._` (as an embedded object into a circe package object) as a first iteration and then extend it from there (if people will start asking about overriding both decoding and encoding). Here is why I think it's not really a problem: most people define endpoints in one places and bootstram them somewhere else so they can use accumulating decoders to define and special-printing to bootstrap.

Reply:
Ok, it makes sense